server/src/controllers/cars.ts

```diff
@@ -14,27 +14,68 @@ import {
   getCarByCustomerId,
 } from '../services';
 import { addCarSchema, updateCarSchema } from '../validation';
+import emailTemplate from '../helpers/emailTemplate';
 
 const addCar = async (req: Request, res) => {
   const { body } = req;
   const { userId } = res.locals.user;
   const data = { ...body, customerId: userId };
   await addCarSchema.validate(body);
   const result = await addCarService(data);
+  const userInfo: { email: string, fullName: string } = await findUserById({ id: userId });
+  if (process.env.NODE_ENV !== 'test') {
+    const emailTitle = 'Your Sell Request Sent Successfully!';
+    const emailBody = `<p>We are happy to tell you that your sell car request has been received with the following details :</p>
+  <ul  class="list">
+     <li>Brand: ${result.brand}</li>
+     <li>Model: ${result.model}</li>
+     <li>Year: ${result.year}</li>
+     <li>Mileage: ${result.mileage}</li>
+     <li>location: ${result.location}</li>
+     <li>price: $${result.price}</li>
+  </ul>
+ <p>Soonly our team will check your request, we will keep in touch with you, you can track your request state through email or using your profile..</p>
+ <button class="button">Go To Profile!</button>`;
```

Comment:
```suggestion
 <button class="button">Go To Profile!</button>`;
```
This will not send me to profile, maybe we can have a link instead